### ISSUE TITLE
Omnichord-esque strumming and inversions

### ISSUE THREAD

**Issue by bhostetler18** [bhostetler18/tonality_features#8]
(These would be new strum bar modes)

**Comment by ghost** [comment]
I think it would also be amazing if we could turn strum mode on by holding the button down
we could switch the mode easily+faster

**Comment by bhostetler18** [comment]
@cazel-ea Sorry a bit confused - which button would you want to hold down on?

**Comment by ghost** [comment]
there is a button for strumming (at least I thought so 😬 ) in chord pads
but if you turn it on, you can only play in strum mode
I guess it'd be better to have this button with hold down option too. this way, we could switch between chords/strums faster
@bhostetler18

**Comment by bhostetler18** [comment]
@cazel-ea Got it - that’s a great idea! I misunderstood what you meant by holding down, but now I realize you mean that strum mode would only be engaged while you actively hold your finger on the button.

**Comment by ghost** [comment]
@bhostetler18 sorry my bad 😊 didn't really know how to explain it
yes but maybe some users need it the way it is now. so, this could be a second option? double tap on the strum button - select fixed strum mode / hold button mode

**Comment by bhostetler18** [comment]
@cazel-ea Yeah I’ll definitely allow both behaviors. I’m thinking of a new, larger button or latch in performance mode which would make it more convenient and less likely to tap the wrong thing.
